subprojects/base-services/src/main/java/org/gradle/internal/Transformers.java

```diff
@@ -98,11 +99,7 @@ public String transform(String input) {
         }
 
         private String wrapWithSingleQuotes(String input) {
```

Comment:
We could consider moving this `wrapWithSingleQuotes` function into TextUtil, too. And add a corresponding `wrapWithDoubleQuotes` function.